src/components/column-header.vue

```diff
@@ -70,7 +70,7 @@
                         </button>
 
                         <column-filter
-                            v-show="props.isOpenFilter === col.field"
+                            v-show="localOpenFilter === col.field"

```

Comment:
@bhaveshpatel200 I dod not change the prop name. The prop is still called `isOpenFilter`. What I did was just create a local variable that stores the props value passed in. 
You were directly changing the value of the prop in line 62 which should not be done.